sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2558,6 +2558,131 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
       }
     }
   }
+
+  test("SPARK-28227: test script transform with aggregation") {
```

Comment:
Could you move the tests into `SQLQueryTestSuite`?

Reply:
> Could you move the tests into `SQLQueryTestSuite`?

This should wait for  https://github.com/apache/spark/pull/29085, since currently we can't use script transform in sql/core

Reply:
Ah, I see.

Reply:
@maropu 
UT have been moved to `transform.sql`